changelogs/unreleased/gh-7932-recovery-mixed-transactions.md

```diff
@@ -0,0 +1,14 @@
+## feature/replication
+
+* Implemented correct recovery of mixed transactions. To do this, set
+  `box.cfg.force_recovery` to `true`. If you need to revert to the old
+  behavior, don't set the `force_recovery` option.
+
+* Note:
+  Let there be two nodes (`node#1` and `node#2`). And let the data be
+  replicated from `node#1` to `node#2`. Suppose that at some point in time,
+  `node#1` is restoring data from an xlog containing mixed transactions. To
+   replicate data from `node#1` to `node#2`, do the following:
+    1. Disable `node#2`
```

Comment:
What do you mean by "disable"?
If I remember correctly, we need to rebootstrap the node. Stop it, remove all the data files from it, and make it rejoin to master.

Reply:
Done